Add release notes for 1.5.6 (#1636)
Automattic/studio: RELEASE-NOTES.txt

Unreleased
==========

1.5.6
=====
* Updated SQLite plugin to 2.2.5 and enabled new AST driver #1506, #1628
* Changed behavior to open the Overview tab after deleting and creating site #1572
* Suppressed punycode warnings for both CLI and the app #1579
* Improved crash screen and fixed a startup crash caused by Studio running from read-only directory #1599
* Added support for sites with custom environment types #1595

> * Set the selective sync checkboxes to be unchecked as the initial state #1626

Reviewer: I think we can also add a line for the improvements in offline mode.

```suggestion
* Set the selective sync checkboxes to be unchecked as the initial state #1626
* Improved offline functionality for site deletion and logout #1637, #1638, #1641
```

Author: Added, thanks.

Unreleased
==========

1.5.6
=====

> * Fixed listing sites with a custom environment type #1595

Reviewer 2: I have noticed we now have two similar records. We could remove this first one.

```suggestion
```

> * Update SQLite plugin to 2.2.5 and enabled new AST driver #1506, #1628

Reviewer 3: ```suggestion
* Updated SQLite plugin to 2.2.5 and enabled new AST driver #1506, #1628
```

I am suggesting to change this to `updated` as another verb says `enabled`

Author: Fixed, thanks.

> * Updated SQLite plugin to 2.2.5 and enabled new AST driver #1506, #1628

Reviewer 4: Now, the version is 2.2.6, I guess.

Author: Good catch, fixed!

* Changed behavior to open the Overview tab after deleting and creating site #1572

> * Update multiple dependencies to the latest versions

Reviewer 3: ```suggestion
* Updated multiple dependencies to the latest versions
```

Now that I see it twice, I am wondering if we have a rule of using `update` in the present tense compared to other items or was this unintentional?

Author: I fixed both - we should keep the convention and use the past tense.

Reviewer 3: Nice, thanks. I thought for a minute maybe there might be a rule for `update` or something  😅

* Suppressed punycode warnings for both CLI and the app #1579

> * Improved crash screen and fixed startup crash from read-only Studio directory #1599

Reviewer 3: ```suggestion
* Improved crash screen and fixed a startup crash caused by a read-only Studio directory.
```
Not a strong preference, feel free to skip

Author: I reworded it slightly, thanks.
